src/components/inputs/localized-multiline-text-input/translation-input.js

```diff
@@ -88,6 +89,11 @@ export default class TranslationInput extends React.Component {
     // amount of lines it should have when collapsed
     const contentExceedsShownRows =
       this.state.contentRowCount > TranslationInput.MIN_ROW_COUNT;
+    const getCommands = props => {
+      if (props.error) return <div>{this.props.error}</div>;
```

Comment:
The function is receiving a `props` argument, but it is using `this.props`. This seems wrong.

Generally, passing plain `props` around is usually a code smell. The function receives too much information. It also stops ESLint from finding issues with the props.

I would recommend to use a `do` expression below instead. See next comment.